scipy/odr/odrpack.py

```diff
@@ -709,6 +712,9 @@ class ODR(object):
     iwork : ndarray, optional
         array to hold the integer-valued working data for ODRPACK. When
         restarting, takes the value of self.output.iwork.
+    overwrite : boolean, optional
```

Comment:
nit: type is `bool` rather than `boolean`

Reply:
Thank you for reviewing. I fixed it.

Reply:
Thank you for reviewing. I fixed it.